charts/scalardb/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and ( .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" ) ( .Values.scalardb.serviceMonitor.enabled ) }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "scalardb.fullname" . }}-metrics
+{{- if .Values.scalardb.serviceMonitor.namespace }}
+  namespace: {{ .Values.scalardb.serviceMonitor.namespace }}
+{{- end }}
+spec:
+  namespaceSelector:
+    matchNames:
+    - {{ .Release.Namespace }}
+  selector:
+    matchLabels:
+      {{- include "scalardb.selectorLabels" . | nindent 6 }}
+  endpoints:
+  - port: 8080
+    interval: {{ .Values.scalardb.serviceMonitor.interval }}
```

Comment:
I think we also need to set `path: /metrics`.
https://github.com/scalar-labs/scalardb/pull/246